src/DisallowFloatAssignedToVariableRule.php

```diff
@@ -26,10 +30,7 @@ public function getNodeType() : string
         return Node::class;
     }
 
-    /**
-     * @return string[]
-     */
-    public function processNode(Node $node, Scope $scope) : array
+    public function processNode(Node $node, Scope $scope) : array // phpcs:ignore SlevomatCodingStandard.TypeHints.TypeHintDeclaration.MissingTraversableReturnTypeHintSpecification
```

Comment:
A type declaration for the type being returned, or an `{@inheritDoc}`, are required